.idea/misc.xml

```diff
@@ -38,7 +38,7 @@
   <component name="ProjectResources">
     <default-html-doctype>http://www.w3.org/1999/xhtml</default-html-doctype>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="21" project-jdk-type="JavaSDK">
```

Comment:
The important part is upgrading the language level to 11, since it doesn't work with 8. The JDK name can be anything, and I'm using Java 21 with language level set to 11.